examples/docs_snippets/docs_snippets/guides/dg/migrating-project/1-tree.txt

```diff
@@ -5,8 +5,8 @@ tree
 ├── my_existing_project
 │   ├── __init__.py
 │   ├── assets.py
-│   └── definitions.py
-├── py.typed
+│   ├── definitions.py
+│   └── py.typed
```

Comment:
hmmmm whats with this

Reply:
oh unrelated -- some random integrity test was failing so I was moving files around trying to fix it, forgot to move this one back. will do